examples/guestbook/README.md

```diff
@@ -107,16 +184,16 @@ Change to the `<kubernetes>/examples/guestbook` directory if you're not already
 
```

Comment:
The instructions incorrectly recommend changing to the examples/guestbook directory. That will break the relative paths in the commands.